tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -1235,7 +1235,7 @@ public function theFollowingUsersShouldBeListedInFollowingGroups(TableNode $tabl
 		$this->featureContext->verifyTableNodeColumns($table, ['username', 'groupname']);
 		$usersGroups = $table->getColumnsHash();
 		foreach ($usersGroups as $userGroup) {
-			$members = $this->listGroupMembers($userGroup['groupname']);
+			$members = $this->featureContext->setResponse($this->listGroupMembers($userGroup['groupname']));
```

Comment:
I don't know if we should set the response in the middle of a loop. Aren't there other alternatives for this?